test/strings/search.jl

```diff
@@ -408,3 +408,13 @@ for T = (UInt, BigInt)
         @test findprev(isletter, astr, T(x)) isa Int
     end
 end
+
+# issue 37280
+let A = [0x40, 0x52, 0x62, 0x52, 0x62]
+    @test findfirst([0x99], A) === nothing
+    @test findfirst([0x52], A) == 2:2
+    @test findfirst([0x52, 0x62], A) == 2:3
+    @test findnext([0x52, 0x62], A, 2) == 2:3
+    @test findnext([0x52, 0x62], A, 3) == 4:5
+    @test findnext([0x52, 0x62], A, 5) === nothing
+end
```

Comment:
Can you also test passing out of bounds indices, both too small and too large? These are easy to get wrong.

Please also check what happens with empty arguments (both).

Reply:
too small fails in other situation:
```
julia> findnext('z', "Hello to the world", -3)
ERROR: BoundsError: attempt to access String
  at index [-3]
```
I will add the > out of bounds test

Reply:
Yeah but you need to test that error to be sure it's there. :-)